docs/docs/segment-owm.md

```diff
@@ -0,0 +1,38 @@
+---
+id: owm
+title: Open Weather Map
+sidebar_label: Open Weather Map 
+---
+
+## What
+
+Shows the current weather of a given location.
+
+**NOTE**: You **must** request an API key at the [Open Weather Map](https://openweathermap.org/price) website.
```

Comment:
You can use the [caution admonition](https://docusaurus.io/docs/markdown-features/admonitions) for this.

Reply:
Added in 3bdacea